Custom AndroidManifest.xml

Unable to use custom AndroidManifest.xml in `/platform/android` (`/app/platform/android` for alloy)

### To Reproduce
1. `ti create -t app --id ti.android.custom.manifest -n ti.android.custom.manifest -p android -u 'http://example.com' -d .`
2. `cd ti.android.custom.manifest`
3. create `platform/android/AndroidManifest.xml`:
```xml
<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android">
        <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="29" />
        <application android:requestLegacyExternalStorage="true">
        </application>
</manifest>
```
4. `ti build -p android --build-only`

```[ERROR] [GRADLE] FAILURE: Build failed with an exception.
[ERROR] [GRADLE] 
[ERROR] [GRADLE] * What went wrong:
[ERROR] [GRADLE] Execution failed for task ':app:processDebugMainManifest'.
[ERROR] [GRADLE] > com.android.manifmerger.ManifestMerger2$MergeFailureException: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 175; Attribute "xmlns:android" was already specified for element "manifest".
[ERROR] [GRADLE] 
[ERROR] [GRADLE] * Try:
[ERROR] [GRADLE] Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
[ERROR] [GRADLE] 
[ERROR] [GRADLE] * Get more help at https://help.gradle.org
[ERROR] [GRADLE] 
[ERROR] [GRADLE] BUILD FAILED in 1
```

### Expected behavior
Custom manifest merged with manifest from `tiapp.xml`

### Environment
Titanium SDK version: 9.3.2.GA, 10.1.0 (master `9d810f06`)
Titanium CLI version: 5.2.2

--- a/android/cli/lib/android-manifest.js
+++ b/android/cli/lib/android-manifest.js
@@ -510,7 +510,7 @@ class AndroidManifest {
 			if (sourceElement.hasAttributes()) {
 				for (let index = 0; index < sourceElement.attributes.length; index++) {
 					const sourceAttribute = sourceElement.attributes.item(index);
-					destinationElement.setAttribute(sourceAttribute.name, sourceAttribute.value);
+					destinationElement.setAttributeNode(sourceAttribute.cloneNode());
 				}
 			}
 

--- a/tests/Resources/ti.app.android.test.js
+++ b/tests/Resources/ti.app.android.test.js
@@ -11,13 +11,17 @@ var should = require('./utilities/assertions');
 
 describe.android('Titanium.App.Android', function () {
 	it('appVersionCode', function () {
+		// Defined in file: ./platform/android/build.gradle
 		should(Ti.App.Android.appVersionCode).not.be.undefined();
 		should(Ti.App.Android.appVersionCode).be.a.Number();
+		should(Ti.App.Android.appVersionCode).be.eql(101);
 	});
 
 	it('appVersionName', function () {
+		// Defined in file: ./platform/android/AndroidManifest.xml
 		should(Ti.App.Android.appVersionName).not.be.undefined();
 		should(Ti.App.Android.appVersionName).be.a.String();
+		should(Ti.App.Android.appVersionName).be.eql('v1.01');
 	});
 
 	it('launchIntent', function () {

--- a/tests/platform/android/AndroidManifest.xml
+++ b/tests/platform/android/AndroidManifest.xml
@@ -1,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionName="v1.01">
+	<application>
+		<meta-data android:name="metatadata1" android:value="Metadata 1"/>
+	</application>
+</manifest>

--- a/tests/platform/android/build.gradle
+++ b/tests/platform/android/build.gradle
@@ -1,0 +1,6 @@
+
+android {
+	defaultConfig {
+		versionCode 101
+	}
+}

